packages/table/src/_docs.scss

```diff
@@ -135,11 +135,11 @@ Styleguide components.table-js.loading
 */
 
 /*
-Cell
+Cell and EditableCell
```

Comment:
What's a better title for this section? It will soon also include `ColumnHeaderCell` and `RowHeader`